module_report/check_modules.py

```diff
@@ -0,0 +1,53 @@
+import importlib
+from pathlib import Path
+import platform
+import subprocess
+
+
+lib_path = Path([*Path('.venv').rglob('site-packages')][0])
+libs = ['PyQt5', 'PyQt6', 'PySide2', 'PySide6']
+qtpy_files = [f.name[:-len('.py')] for f in Path('qtpy').glob('*.py') if f.name.startswith('Qt')]
+
+
+modules = {}
+
+
+for lib in libs:
+    path = lib_path / lib
+    files = path.glob('*.pyi')
+    for f in files:
+        if f.name.startswith('Qt'):
+            name = f.name[:-len('.pyi')]
+            if name not in modules:
+                modules[name] = {'libs':[], 'missing': name not in qtpy_files}
+            modules[name]['libs'].append(lib)
```

Comment:
```suggestion
for api_name in qtpy.API_NAMES.values():
    try:
        api_path = pkgutil.resolve_name(api_name).__path__
    except ImportError:
        continue
    api_modules = [mod.name for mod in pkgutil.iter_modules(api_path)]
    api_modules =  [mod for mod in api_modules if mod.name.startswith('Qt')]
    for module in api_modules:
        if module not in QT_MODULES and module not in FALSE_POSITIVES:
            modules[module] = {'apis':[], 'missing': name not in QTPY_MODULES}
        modules[api_module]['apis'].append(module)
```

As discussed above, instead of this hacky, fragile approach (particularly so since the previous was relying on internal implementation details of their _static type stubs_, which could easily change at any time) you can use `pkgutil`.

Also, move the false positive check here; its not clear why it was ex post facto before.

Reply:
@DaelonSuzuka Not sure if you noticed, but I made a mistake here when refactoring this. This line:

```python
    api_modules =  [mod for mod in api_modules if mod.name.startswith('Qt')]
```

should be

```python
    api_modules =  [mod for mod in api_modules if mod.startswith('Qt')]
```